src/data/faq.json

```diff
@@ -723,7 +723,7 @@
                             "With the introduction of Corona-Warn-App version 1.9, a new version of the Exposure Notification Framework (ENF) from Apple/Google is used, version 2.0. The data from this interface is used as the basis for calculating the individual risk.",
                             "While the interface's first version 1.0 used average attenuation values for all encounters with another device during the course of the day, the new version 2.0 now considers 30-minute time windows.",
                             "Each of these windows must meet certain criteria to be considered:",
-                            "<ul><li>The signal attenuation must have been below 73dB for at least 10 minutes within the 30-minute window</li>",
+                            "<ul><li>The signal attenuation must have been below 73dB for at least 10 minutes (reduced to 5 minutes on Feb 23, 2021, see <a href='../blog/2021-02-23-corona-warn-app-risk-calculation-optimization/' target='_blank' rel='noopener noreferrer'>blog post</a>) within the 30-minute window</li>",
                             "<li>The transmission risk level must be at least III (3). It is determined when sharing the diagnosis codes per day according to defined criteria such as symptom status and/or symptom onset</li></ul>",
                             "If these criteria are <b>not</b> met, an encounter is <b>not count</b> within the app. It is a 'non-risk encounter'.  However, in the operating system's contact log, a count is performed. However, this is not an error.",

```

Comment:
<b>not count</b>
should be
<b>not counted</b>

Reply:
thanks